bin/init_repo.sh

```diff
@@ -77,10 +79,11 @@ for template in "${templates[@]}" ; do
         done
         echo "${result}" > "${dest_path}"
         printf "${green}Processed and created ${nc}%s\n" "${dest_path}"
-        if [ "${remove_templates}" = "Y" ] || [ "${remove_templates}" = "y" ]; then
-          rm "${source_path}"
-        fi
     else
         printf "${red}File not found ${nc}%s\n" "${source_path}"
     fi
 done
+
+if [ "${remove_templates}" = "Y" ] || [ "${remove_templates}" = "y" ]; then
+    rm -rf "templates"
+fi
```

Comment:
I realized that with the previous setting of this option, the `templates` folder with its "python versions" subfolders were kept also when remove_templates was set to 'Y'. I think that this new behaviour is the wanted one